src/main/java/xyz/thepathfinder/android/Cluster.java

```diff
@@ -156,13 +160,21 @@ protected static Cluster getInstance(JsonObject clusterJson, PathfinderServices
      * @return Whether the json can be parsed to a cluster.
      */
     private static boolean checkClusterFields(JsonObject clusterJson) {
-        return clusterJson.has("path") &&
+        //TODO revert after path update
```

Comment:
Is there an open issue for Dan about this? If so can you link this PR to that issue?